test/controllers/content_items_controller_test.rb

```diff
@@ -72,6 +72,41 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_redirected_to content_item['base_path']
   end
 
+  test "returns HTML when an unspecific accepts header is requested (eg by IE8 and below)" do
+    request.headers["Accept"] = "*/*"
+    content_item = content_store_has_schema_example('travel_advice', 'full-country')
+
+    get :show, params: {
+      path: path_for(content_item)
+    }
+
+    assert_match(/text\/html/, response.headers['Content-Type'])
+    assert_response :success
+    assert_select '#wrapper'
+  end
+
+  test "returns a 406 for XMLHttpRequests" do
```

Comment:
(Aside: Passing an empty `Accepts: ` header in minitest appears to get ignored.)